core/src/main/java/io/questdb/griffin/engine/functions/math/CeilingFloatFunctionFactory.java

```diff
@@ -33,34 +33,12 @@
 import io.questdb.griffin.engine.functions.UnaryFunction;
 import io.questdb.std.IntList;
 import io.questdb.std.ObjList;
```

Comment:
Would be nice to remove unused imports here.

Reply:
yeah I overlooked them, gimme a min